activerecord/test/cases/fixtures_test.rb

```diff
@@ -711,33 +712,6 @@ def test_this_should_run_cleanly
   end
 end
 
-class FixturesBrokenRollbackTest < ActiveRecord::TestCase
```

Comment:
This test removal reflects slightly simplified behavior:

Prior to this PR, if the case of a fixture-load process failure (simulated here by stubbing `load_fixtures` with `raise`), then the test transaction would not begin, and so this test ensures that a rollback must not be attempted in the fixture-teardown step when the transaction is not active.
With this PR the fixture-load and transaction-wrapping processes are no longer tightly coupled, so this edge case is no longer relevant.